fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/impl/datagen/FabricDataGenHelper.java

```diff
@@ -123,13 +129,22 @@ private static void runInternal() {
 				final String effectiveModId = entrypoint.getEffectiveModId();
 				ModContainer modContainer = entrypointContainer.getProvider();
 
+				HashSet<String> keys = new HashSet<>();
+				entrypoint.addJsonKeySortOrders((key, value) -> {
+					jsonKeySortOrders.put(key, value);
```

Comment:
Might want to check for null keys here to prevent accidental bugs

Reply:
What would be more preferred here? Throwing an exception or ignoring the value and logging a warning?

Reply:
User error so exception. You can just add `Objects.requireNonNull` at the top of the lambda. (See other modules it's used a lot)